pkg/errors/errors_test.go

```diff
@@ -0,0 +1,65 @@
+package errors
+
+import (
+	"errors"
+	"net/http"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+func TestErrMsg(t *testing.T) {
+	const op Op = "TestErrMsg"
+	msg := "test error"
+	err := E(op, msg)
+	require.Equal(t, msg, err.Error())
+}
+
+func TestErrEmbed(t *testing.T) {
+	const op Op = "TestErrEmbed"
+	msg := "test error"
+	childErr := errors.New(msg)
+	err := E(op, childErr)
+	require.Equal(t, err.Error(), childErr.Error())
+}
+
+func TestSeverity(t *testing.T) {
+	const op Op = "TestSeverity"
+	msg := "test error"
+
+	err := E(op, msg)
+	require.Equal(t, logrus.ErrorLevel, Severity(err))
+
+	err = E(op, msg, logrus.WarnLevel)
+	require.Equal(t, logrus.WarnLevel, Severity(err))
+
+	err = E(op, err)
+	require.Equal(t, logrus.WarnLevel, Severity(err))
```

Comment:
do we need to check the warning level is set twice? or was this meant to check another logging level?

Reply:
@robjloranger yes this is meant to check only the bottom level, this is because when the parent doesn't have a Severity, we take the Severity of the child.